components/layout/Layout.tsx

```diff
@@ -95,7 +94,7 @@ const Layout = ({ children, toggleTheme, isDarkTheme }: LayoutProps) => {
   };
 
   return (
-    <div>
+    <header>
```

Comment:
The `div` here is a container for the whole app, so it will have to stay as an enclosing `div`.

The top section (inside `Wrapper`) is indeed the header, then inside the `Main` tags is where the main part of the app get dynamically rendered, and finally the `Footer` component at the bottom tells Next to render the Footer on every page.

The `header` change would be perfect inside the `Wrapper` component. 
`components > layout > styles.ts` line 4. The `Wrapper` would just need to be based on a `header` instead of a `div`